mock/kswg_mock/routers/kubeseal.py

```diff
@@ -92,7 +132,23 @@ def encrypt_value_or_file(
         valid_k8s_name(secret_name),
         "--cert",
         cert,
+        "--scope",
```

Comment:
The options for `namespace` and `name` above should be removed. We add them later in the process.